scripts/rebuild-deb.sh

```diff
@@ -58,7 +58,7 @@ echo "------------------------------------------------------------"
 mkdir -p "${BUILDDIR}/usr/local/bin"
 cp ./default/src/app/cli/src/coda.exe "${BUILDDIR}/usr/local/bin/coda"
 cp "${SCRIPTPATH}/../src/app/libp2p_helper/result/bin/libp2p_helper" "${BUILDDIR}/usr/local/bin/coda-libp2p_helper"
-patchelf --set-interpreter /lib64/ld-linux-x86-64.so.2 "${BUILDDIR}/usr/local/bin/coda-libp2p_helper"
+sudo patchelf --set-interpreter /lib64/ld-linux-x86-64.so.2
```

Comment:
don't you still need the libp2p helper file argument?

Reply:
@psteckler Can you give an example argument, I have less context with what this is doing

Reply:
@yourbuddyconner I thought the required change was to add `sudo`, but the actual file for the interpreter was omitted in the command (the `$BUILDDIR)/usr....` file).

Reply:
I don't think we can use `sudo` during a build. Is it demanded here?

Reply:
good catch, yes.

Reply:
@bkase I think it was a dangling symlink error, not a permission error, so no.